frontend/style/filterpage.css

```diff
@@ -0,0 +1,18 @@
+.FilterPage {
+  background-color: black;
+  opacity: 0.5;
+  width: 100%;
+  height: 100vh;
+  position: absolute;
+  z-index: 1001;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.button_filter_close {
```

Comment:
Keep your coding conventions, all of your css classes are camelCase named. Rename it in buttonFilterClose.
Remember to change it in jsx too.